onnxruntime/core/providers/shared_library/provider_bridge_provider.cc

```diff
@@ -239,3 +312,172 @@ std::unique_ptr<OpKernelInfo> CopyOpKernelInfo(const OpKernelInfo& info) {
 }
 
 }  // namespace onnxruntime
+
+#include "core/providers/cpu/tensor/unsqueeze.h"
```

Comment:
What's the reason to include the operator headers here?

Reply:
No reason now, I forget if there originally was one. I moved them to the top and made them more alphabetical

Reply:
I meant why do we need to include the operator headers in this bridge code?